peps/pep-0101.rst

```diff
@@ -562,14 +562,10 @@ the main repo.
   - Make sure the ``SOURCE_URI`` in ``Doc/tools/extensions/pyspecific.py``
     points to ``main``.
 
-  - Update the version numbers for the Windows builds in PC/ and
-    PCbuild/, which have references to python38.
-    NOTE, check with Steve Dower about this step, it is probably obsolete.::
-
-        $ find PC/ PCbuild/ -type f | xargs sed -i 's/python38/python39/g'
-        $ git mv -f PC/os2emx/python38.def PC/os2emx/python39.def
-        $ git mv -f PC/python38stub.def PC/python39stub.def
-        $ git mv -f PC/python38gen.py PC/python39gen.py
+  - Update the version numbers for the Windows builds
+    which have references to ``python38``::
+
+        ls PC/pyconfig.h.in PCbuild/rt.bat | xargs sed -i 's/python3\.\?[0-9]\+/python39/g'
```

Comment:
The `sed` isn't going to work - the replacement needs to preserve the optional `.`

I expect with only two files it'll be easier to do them manually anyway, but having a command ready to copy-paste here is going to be awfully tempting.

Reply:
Yep, pushed a new one!